src/compiler/crystal/semantic/call_error.cr

```diff
@@ -90,6 +90,9 @@ class Crystal::Call
       error_msg = String.build do |msg|
         if obj && owner != mod
           msg << "undefined method '#{def_name}' for #{owner}"
+        elsif logical_operator?(def_name)
+          msg << "undefined method '#{def_name}'"
+          similar_name = convert_def_name_to_logical_operator(def_name)
```

Comment:
Note that this shouldn't always be suggested, only when there's no receiver. Doing that in line 88 will always suggest it, which is not very good.